src/components/dataset/query/QueryView.jsx

```diff
@@ -99,6 +107,15 @@ export class QueryView extends React.PureComponent {
           PAGE_SIZE,
         ),
       );
+      const bq = new BigQuery();
+      bq.getTotalRows(
+        dataset.dataProject,
+        `#standardSQL
+          SELECT COUNT(*) FROM \`${dataset.dataProject}.datarepo_${dataset.name}.${selected}\` AS ${selected}
+          ${joinStatement}
+          ${filterStatement}`,
+        token,
+      ).then((response) => dispatch(countResults(parseFloat(response))));
```

Comment:
i would use `parseInt` here instead since this isn't going to be a decimal number